### DISCUSSION TITLE
Add description to all "allOf" array types in renovate.json schema?

### DISCUSSION THREAD

**Discussion by japj** [renovatebot/renovate#8979] [Request Help] [answered]
In order to properly document the contents of the renovate.json, would it be possible to extend the schema of renovate.json to include a description property for all "allOf" array types?

e.g. I want a short description/comment for each of my packageRules, but as far I can tell that is currently not possible due to the description missing from the schema and comments not allowed in the json format

**Comment by viceice** [comment, marked as answer]
even if `description` is missing in schema, most obejects are open, so you can simply add `description` and it should validate.
